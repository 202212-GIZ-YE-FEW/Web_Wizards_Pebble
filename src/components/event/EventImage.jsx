import Image from "next/image";

import Wave from "~/event/Wave.png";

const EventImage = ({ imageUrl }) => {
    return (
        <div className='order-1 w-full lg:w-1/2 flex-shrink-0'>
            <Image
                priority={true}
                id='sitting-img'
                src={imageUrl ? imageUrl : Wave}
                width={720}
                height={572}
                alt='Sitting or a rock'
                className=' mx-auto  mt-16'
            />
        </div>
    );
};

export default EventImage;
