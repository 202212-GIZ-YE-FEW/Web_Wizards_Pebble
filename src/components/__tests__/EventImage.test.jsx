import renderer from "react-test-renderer";

import EventImage from "../event/EventImage";

it("renders correctly", () => {
    const tree = renderer.create(<EventImage />).toJSON();
    expect(tree).toMatchSnapshot();
});
