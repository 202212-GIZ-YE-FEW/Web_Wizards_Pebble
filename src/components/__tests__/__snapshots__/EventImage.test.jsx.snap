// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="order-1 w-full lg:w-1/2 flex-shrink-0"
>
  <img
    alt="Sitting or a rock"
    className=" mx-auto  mt-16"
    data-nimg="1"
    decoding="async"
    height={572}
    id="sitting-img"
    onError={[Function]}
    onLoad={[Function]}
    src="/_next/image?url=%2Fimg.jpg&w=1920&q=75"
    srcSet="/_next/image?url=%2Fimg.jpg&w=750&q=75 1x, /_next/image?url=%2Fimg.jpg&w=1920&q=75 2x"
    style={
      Object {
        "color": "transparent",
      }
    }
    width={720}
  />
</div>
`;
